packages/react/src/Icon/Icon.tsx

```diff
@@ -1,33 +1,81 @@
 import React, {type SVGAttributes} from 'react'
 import styles from './Icon.module.css'
 import clsx from 'clsx'
-import {Icon as OcticonProps} from '@primer/octicons-react'
+import {type Icon as OcticonProps} from '@primer/octicons-react'
 import {Colors} from '../constants'
 
-/**
- * Design Tokens
- */
 import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/icon/colors.css'
 
-export const IconColors = Colors
-export const defaultIconColor = IconColors[0]
+export const namedIconSizes = ['small', 'medium', 'large'] as const
+export type NamedIconSize = (typeof namedIconSizes)[number]
+
+export const numericIconSizes = [12, 16, 20, 24, 28, 32, 36, 40, 44, 48, 52] as const
```

Comment:
Can we max out at `44` please [per this guidance
](https://github.com/primer/brand/issues/773#issuecomment-2429832267)

```suggestion
export const numericIconSizes = [20, 24, 28, 32, 36, 40, 44] as const
```

Reply:
Happy to do that if @danielguillan is happy.

These numbers have come from [Dani's FIgma designs](https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/branch/9CEUShb0wiFWbpQYpQMdpk/Primer-Brand?node-id=15397-35&node-type=canvas&t=DEcuCT9yVFeK8Aua-0)

Reply:
Ah cool, that's good to know. We'll need to follow up In Figma after too, as the Site guidance supersedes that. Particularly in the React version, we'd want to provide the correct guardrails so recommend we go ahead with the 20-44 range.

Reply:
That's updated

Reply:
I'm happy to limit the scale and expand it later if needed. I just want to check with @simmonsjenna to confirm that we've considered scenarios beyond Bento, Card, and Pillar in making this decision. For example, I'm thinking about icons that align with the default body text (16px) in Primer Brand components like Button, Label, and UnorderedList, among others. I'm not suggesting we necessarily use Icon in those, but those are potential use cases. Also custom elements might benefit from the expanded scale as there are several places in dotcom where use icons smaller than 20 and larger than 44.

Reply:
Sounds good!

I believe there's only one icon larger than 44 in landing pages but [there are some more](https://github.com/search?q=org%3Agithub++%2F%3C.*Icon+.*size%3D%5C%7B%28%5B4-9%5D%5Cd%7C%5Cd%7B3%2C%7D%29%5C%7D%2F&type=code&p=1) on product pages that could potentially use Primer Brand components at some point (e.g., upsell). That said, we don't need to support those larger sizes if we want to discourage that. Octicons aren't actually designed for larger sizes either way. We can create some guardrails around that.